.github/workflows/test-course-definition.yml

```diff
@@ -143,6 +143,21 @@ jobs:
 
       - run: find courses/${{github.event.repository.name}} -name '*.rs' | xargs rustfmt --edition "2021" --check
 
+  docker_lint:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+
+    env:
+      HADOLINT_RECURSIVE: "true"
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: hadolint/hadolint-action@v3.1.0
+        with:
+          dockerfile: "*Dockerfile"
+          recursive: true
```

Comment:
Might be useful to add a quick comment here mentioning why this rule is disabled. Something like: 

```
# Disabled rules: 
# - DL3059: We split steps to stream logs independently for different steps
```